benches/osrank_naive.rs

```diff
@@ -0,0 +1,164 @@
+#![macro_use]
+extern crate criterion;
+extern crate rand;
+extern crate rand_xorshift;
+
+use criterion::{Criterion, Benchmark};
+use criterion::criterion_group;
+use criterion::criterion_main;
+use osrank::protocol_traits::graph::{Graph, GraphObject};
+use osrank::protocol_traits::ledger::{MockLedger, LedgerView};
+use osrank::types::{Artifact, ArtifactType, Dependency, DependencyType, Network, Osrank, Weight};
+use osrank::algorithm::osrank_naive;
+use osrank::algorithm::random_walk;
+use osrank::algorithm::rank_network;
+use osrank::importers::csv::import_network;
+use num_traits::Zero;
+use rand_xorshift::XorShiftRng;
+use crate::rand::SeedableRng;
+use std::fs::File;
+
+type MockNetwork = Network<f64>;
```

Comment:
We are starting to use these alias in enough places perhaps to warrant its move into some sort of helper module at the library level? (or even just in `types.rs`?)

Reply:
Will create a task for this to speed up merging.